potassium/potassium.py

```diff
@@ -74,19 +88,36 @@ def __init__(self, name):
         self.name = name
 
         # default init function, if the user doesn't specify one
-        self._init_func = lambda: {}
+        self._init_func = lambda _: {}
         # dictionary to store unlimited Endpoints, by unique route
         self._endpoints = {}  
         self._context = {}
-        self._gpu_lock = Lock()
-        self._background_task_cv = Condition()
-        self._sequence_number = 0
-        self._sequence_number_lock = Lock()
-        self._idle_start_time = 0
-        self._last_inference_start_time = None
         self._flask_app = self._create_flask_app()
+        self._event_queue = ProcessQueue()
+        self._response_queue = ProcessQueue()
+        self._response_mailbox = ResponseMailbox(self._response_queue)
+
+        self._num_workers = int(os.environ.get("POTASSIUM_NUM_WORKERS", 1))
```

Comment:
should this be in a config file? not sure how many envs there are in the project.